src/main/java/gregtech/api/metatileentity/MTETrait.java

```diff
@@ -39,6 +43,26 @@ public NBTTagCompound serializeNBT() {
     public void deserializeNBT(NBTTagCompound compound) {
     }
 
+    @SuppressWarnings("static-method")
+    protected boolean isConfigurable() {
+        return false;
+    }
+
+    @Override
+    public ResourceLocation getConfigurationID() {
+        throw new AssertionError("unused");
```

Comment:
Wouldn't it be better to just not implement `IConfigurable` for classes that... don't implement it?

Reply:
I did it this way, so it is easy for me to the refactor while it is being developed. 

I don't intend for MTETrait or the item filter handler to really implement that interface since they don't expose the capability directly.